roles/matrix-synapse/templates/synapse/systemd/matrix-synapse-worker@.service.j2

```diff
@@ -0,0 +1,29 @@
+#jinja2: lstrip_blocks: "True"
+# c.f. https://github.com/matrix-org/synapse/pull/4662
+[Unit]
+Description=Synapse Matrix Worker
+After=matrix-synapse.service
+BindsTo=matrix-synapse.service
+
+[Service]
+Type=simple
+
+# Intentional delay, so that the homeserver (we likely depend on) can manage to start.
+ExecStartPre=/bin/sleep 5
+
+# systemd ftw 🤦‍♂️
+# https://github.com/systemd/systemd/issues/14895#issuecomment-594123923
+ExecStart=/bin/sh -c "WORKER=%i; WORKER=$${WORKER%%:*}; \
+			exec /usr/bin/docker exec \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			matrix-synapse \
+			python -m synapse.app.$${WORKER} -c /data/homeserver.yaml -c /data/worker.%i.yaml"
+
+ExecStop=/usr/bin/docker exec matrix-synapse pkill -f %i
```

Comment:
I can't stop a worker service with systemd because matrix-synapse container hasn't pkill command. 

`/usr/bin/docker exec matrix-synapse pkill -f 18082
OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"pkill\": executable file not found in $PATH": unknown
`

Reply:
This is not related to this Pull Request. It's a general issue affecting the whole playbook, because of Synapse's recent (in v1.18.0) switch from an Alpine to a Debian base. See: https://github.com/matrix-org/synapse/pull/7839

Reply:
Oh, I'm sorry.. I did not look into that much. We only use this `pkill` thing here. It is indeed an issue that only affects the workers setup.

Reply:
This has been redone. Should be fixed.